.gitignore

```diff
@@ -1,2 +1,3 @@
+/.idea/
```

Comment:
IMHO the settings folder of your IDE such as `.idea` for PHPStorm or `netbeans` for NetBeans should always be included as part of your Global Git Ignore but never in a project's Git Ignore file.

Reply:
I agree, although PHPStorm has established itself as a defacto standard.

Reply:
I had a few weeks ago a discussion about IDEs with a colleague that basically said he has a love/hate relationship with PHPStorm. Also, until a few months ago, I was working with another developer who moved back to Netbeans due to PHPStorm was ”too slow”.

Although I love PHPStorm, I understand that not everyone wants to use it.